tools/scripts-core/package.json

```diff
@@ -18,7 +18,7 @@
     "url": "https://github.com/Talend/ui.git"
   },
   "scripts": {
-    "lint": "talend-scripts lint ./",
+    "lint": "talend-scripts lint ./scripts/",
```

Comment:
no lint in utils folder?

Reply:
I have moved all file under src so I can use without arguments